src/composables/node/useNodePricing.ts

```diff
@@ -355,7 +355,12 @@ const apiNodeCosts: Record<string, { displayPrice: string | PricingFunction }> =
         const modelValue = String(modelWidget.value)
 
         // Same pricing matrix as KlingTextToVideoNode
-        if (
+        if (modelValue.includes('v2-5-turbo')) {
+          if (durationValue.includes('10')) {
+            return '$0.7/Run'
+          }
+          return '$0.35/Run' // 5s default
```

Comment:
Users can't see this comment. Do we hint at length per run anywhere else?

Reply:
yes, we do that in every node that support length